src/index.js

```diff
@@ -9,9 +9,21 @@ if (globalThis.fetch && globalThis.Headers && globalThis.Request && globalThis.R
   }
 } else {
   module.exports = {
+    /**
+     * @type {typeof import("node-fetch/index").default }
+     */
```

Comment:
Is the `@type` annotation necessary?  Can tsc not figure it out form the assignment from `require`?

Reply:
it was when I was using the fork but now it is irrelevant. Forgot to remove 🤦